pkgs/top-level/python2-packages.nix

```diff
@@ -12,7 +12,7 @@ with self; with super; {
     inherit (pkgs) antlr4;
   };
 
-  appleseed = toPythonModule (pkgs.appleseed.override { inherit (self) python; });
+  appleseed = disabled super.appleseed;
```

Comment:
If it is always disabled we can also remove it.

Reply:
It is *not* disabled on python3. This is (as far as I can tell) the way it was suggested to me that I disable a package only on python2 in  #148329 (based on the existing examples in this file).

Reply:
This should be done via disabled in the package itself. This file should only contain python2 only packages or entries when we load older versions via an other file.

Reply:
For python2, I think this is fine

Reply:
we are trying to get rid of it for 22.05